README.md

```diff
@@ -26,14 +31,31 @@ You can run `locustor` with two modes: `run` and `compare`.
 ### Run
 
 When executed with `run`, `locustor` will execute the tests, output the
-results as csv files and print the information in the screen
+results as csv and json files in your directory selected.
 
-#### Change default use cases
+## Configuration
 
-#### Select output directory
+**Locustor** *(host, locust_file,  work_dir, num_clients,  hatch_rate, run_time)*
+
+host= Host to test
+locust_file= Locust file location
+work_dir= Directory where to leave the result
+num_clients=Num client tests(int)
+hatch_rate=Hatch rate test(int)
+run_time = Test execution time ej:'60s'
```

Comment:
* `host`: Host to test

Reply:
a que te refieres?

Reply:
Es una propuesta de formato para el readme

```
* `host`: Host to test
```